resources/product-listing.js

```diff
@@ -20,11 +20,35 @@ function ProductListing(shopify) {
 }
 
 assign(ProductListing.prototype, omit(base, [
-  'create',
-  'delete',
   'update'
 ]));
 
+/**
+ * Adds the productId to a sales channel
+ *
+ * @param {Number} productId ProductID to add to the sales channel
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+ProductListing.prototype.create = function addProductId(productId) {
+  const url = this.buildUrl(`${productId}`);
+  return this.shopify.request(url, 'PUT', undefined, {})
+    .then(body => body[this.key]);
+};
+
+/**
+ * Deletes the productId to a sales channel
+ *
+ * @param {Number} productId ProductID to delete from the sales channel
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+ProductListing.prototype.delete = function deleteProductId(productId) {
```

Comment:
I think we can remove this as the default implementation https://github.com/MONEI/Shopify-api-node/blob/53b6ac8c1253a46aa423f415bb0c3f7bdd80df95/mixins/base.js#L45-L48 should work.